adagrams/game.py

```diff
@@ -1,11 +1,79 @@
+import random
+from collections import Counter
+LETTER_POOL20 = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
 def draw_letters():
-    pass
+    values = list(LETTER_POOL20.values())
+    keys = list(LETTER_POOL20.keys())
+    players_hand =random.sample(keys, counts = values, k =10)
+    return players_hand
 
 def uses_available_letters(word, letter_bank):
-    pass
-
+    word = word.upper()
+    available_bank = Counter(letter_bank)
+    for letter in word:
+        if letter in letter_bank and available_bank[letter]:
+            available_bank[letter] -= 1
+            available = True
```

Comment:
Instead of creating a variable called `available` and returning it on line 46, you could just have line 46 directly return True (and omit available on line 43), doing so would make it so the different branches of execution for this method look consistent -- you return False in the else case and at the end of looping you return True. 

The way you have it written does work, but it's kind of funny to see one condition return False and another condition return `available` and then have to go back to see what `available` is.